### DISCUSSION TITLE
Filtering of iteration results

### DISCUSSION THREAD

**Discussion by logerse** [rizinorg/rizin#4674] [Q&A]
Is there any way to apply a filter to the result of an iteration without using intermediate files?
Expectation: ``` axt @@=`izz~Something[0]`~[0]:..2```
Reality: ```axt @@=`izz~Something[0]` > /tmp/temp.1; cat /tmp/temp.1~[0]:..2```

**Comment by XVilka** [comment]
I think it's actually an issue/missing feature.
